src/api/app/assets/stylesheets/webui/application/bento/base.scss

```diff
@@ -96,38 +91,112 @@ blockquote{border-left:5px solid #690;color:#666;font-style:italic;}
+        &:focus {
+            color:#330;
+            box-shadow:none;
+        }
+    }
+}
+#subheader {
+    font-size:0.8em;
+    margin-top:-15px;
+    min-height:17px;
+    a {
+        text-decoration:none;
+    }
+}
+#breadcrump {
+    font-size:0.9em;
+    color:#333;
+    img{
+        float:left;
+        margin-right:5px;
+        border-bottom:1px solid transparent;
+    }
+    a {
+        color:#666;
+        &:hover, &:focus {
+            text-decoration:underline;
+        }
+    }
+}
 #login-wrapper{text-align:right;height:1em;}
 .main ul, .main ol{list-style-position:outside;}
 .main li{padding-left:0;}
-#login-form{display:none;width:250px;height:auto;z-index:3;font-size:0.7em;position:absolute;top:31px;right:0;padding-bottom:10px;color:#f6f6f6;background:#333 image-url('images/gradient-darker.png') repeat-x scroll bottom left;border-radius:0 0 10px 10px;border-width:1px;border-style:solid;border-color:transparent #333 #111;opacity:0.95;-moz-opacity:0.95;-webkit-box-shadow:0 1px 3px #999;-moz-box-shadow:0 1px 3px #999;box-shadow:0 1px 3px #999;}
-#login-form input{width:208px;font-size:1.5em;padding:2px 3px;border:3px solid #222;-moz-border-radius:5px;-webkit-border-radius:5px;-khtml-border-radius:5px;border-radius:5px;}
-#login-form input:focus{color:#39c;border-color:#39c;-webkit-box-shadow:0 1px 3px #222;-moz-box-shadow:0 1px 3px #222;box-shadow:0 1px 3px #222;}
-#login-form input[type="submit"]{border-color:#f6f6f6;-webkit-box-shadow:0 0px 2px #000;-moz-box-shadow:0 1px 1px #111;box-shadow:0 1px 1px #111;color:#eee;cursor:pointer;background:#9f3 image-url('images/gradient-dark-nav.png') repeat-x scroll 0 -15px;font-weight:bold;text-shadow:0 0px 3px #333;width:220px;}
-#login-form input[type="submit"]:hover{border-color:#39c;}
-#login-form p{margin:15px;clear:left;position:relative;}
-#login-form label{display:block;position:absolute;top:8px;left:7px;color:#666;width:100px;font-size:13px;}
-#login-form .slim-footer{text-align:center;margin-bottom:0;}
-#login-form .slim-footer a{text-decoration:none;color:#eee;font-weight:bold;}
-#login-form .slim-footer a:hover{text-decoration:underline;}
-#local-user-actions{}
+#login-form {
+    display:none;
+    width:250px;
+    height:auto;
+    z-index:3;
+    font-size:0.7em;
+    position:absolute;
+    top:31px;
+    right:0;
+    padding-bottom:10px;
+    color:#f6f6f6;
+    background:#333 image-url('images/gradient-darker.png') repeat-x scroll bottom left;
+    border-radius:0 0 10px 10px;
+    border-width:1px;
+    border-style:solid;
+    border-color:transparent #333 #111;
+    opacity:0.95;
+    box-shadow:0 1px 3px #999;
+
+    input{width:208px;font-size:1.5em;padding:2px 3px;border:3px solid #222;border-radius:5px;}
+    input:focus{color:#39c;border-color:#39c;box-shadow:0 1px 3px #222;}
+
+    .slim-footer{text-align:center;margin-bottom:0;}
+    .slim-footer a{text-decoration:none;color:#eee;font-weight:bold;}
+    .slim-footer a:hover{text-decoration:underline;}
```

Comment:
Sure, I will update the PR to use this style for more of the rules. Wanted to get feedback first :+1: